File: Task 1/Java/app/src/main/java/dev/ogabek/java/MainActivity.java
package dev.ogabek.java;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        initViews();

    }

    private void initViews() {
        Button start, stop;
        TextView status;

        start = findViewById(R.id.btn_startService);
        stop = findViewById(R.id.btn_stopService);

        status = findViewById(R.id.tv_serviceStatus);

        start.setOnClickListener(view -> {
            startService(new Intent(getApplicationContext(), MyIntentService.class));
            status.setText("Service is Running");
        });

        stop.setOnClickListener(view -> {
            MyIntentService.stopService();
            status.setText("Service is Stopped");
        });

    }
}

File: Task 1/Kotlin/app/src/main/java/dev/ogabek/kotlin/MainActivity.kt
package dev.ogabek.kotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Button
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        initViews()

    }

    private fun initViews() {
        val start: Button = findViewById(R.id.btn_startService)
        val stop: Button = findViewById(R.id.btn_stopService)
        val status: TextView = findViewById(R.id.tv_serviceStatus)

        start.setOnClickListener {
            Intent(this, MyIntentService::class.java).also {
                startService(it)
                status.text = "Intent Service Started"
            }
        }

        stop.setOnClickListener {
            MyIntentService.stopService()
            status.text = "Intent Service Stopped"
        }

    }
}